.github/workflows/run-backend-tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: true
       matrix:
         go:
-          - 1.19
+          - '1.20.x'
```

Comment:
Do we need the `.x` or can we leave this as just `1.20`?

Reply:
@aangelisc AFAICT they do do different things, `1.20` locks to the minor version, but `1.20.x` gets the latest patch version. Would we prefer that or do we just want to get the latest patch vesion?

Reply:
My gut tells me we should lock to the version Grafana uses but I'm open to suggestions

Reply:
According to the `go.mod` in Grafana, the version is `1.20` but in the case of this file, it would get the latest patch version. In the GitHub Actions config, they are using `1.20.6` or `>=1.20` (which is a wider range than `1.20.x`).

Reply:
Okay lets do it😊